src/pages/Dashboard/ChildComponent.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const ChildComponent = ({ heading, button, onMouseHover }) => {
```

Comment:
Please rename this component with a proper well-defined name?

Reply:
done